src/network/tc/tbf.c

```diff
@@ -304,13 +304,13 @@ static int token_bucket_filter_verify(QDisc *qdisc) {
 
         if (tbf->limit > 0 && tbf->latency > 0)
                 return log_warning_errno(SYNTHETIC_ERRNO(EINVAL),
-                                         "%s: Specifying both LimitSize= and LatencySec= is not allowed. "
+                                         "%s: Specifying both LimitBytes= (LimitSize=) and LatencySec= is not allowed. "
```

Comment:
Please drop the deprecated option in the log message.
We usually drop deprecated options from man pages, and they are silently supported for backward compat. Hence, we should not log them.